src/controllers/deploy.go

```diff
@@ -30,7 +30,8 @@ func deploy(params *deployAction) {
 
 	writeErr := helper.WriteToFile(path.Join(logDir, logPath), string(output))
 	if writeErr != nil {
-		log.Errorf("An error occured while writing to %s: %v", path.Join(logDir, logPath), writeErr)
+		log.Errorf("An error occured while writing to %s: %s", path.Join(logDir, logPath), writeErr.Error())
+		slack.PostChatMessage(channel, fmt.Sprintf("Could not write to %s error: %s\n", path.Join(logDir, logPath), writeErr.Error()), nil)
```

Comment:
Just make the error message have the `actionLog` too (like https://github.com/devclub-iitd/DeployBot/blob/6b8ffa26276d5c54d5f8882f395c67445174ae75/src/controllers/deploy.go#L41) so there would some context.